recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: libgpuarray-{{ version }}.tar.gz
   url: https://github.com/Theano/libgpuarray/archive/v{{ version }}.tar.gz
-  sha256: 9434bc9d3c76da07231a6fd4fed87f5e08b0c5dc978a9f93db2233739c6ae567
+  sha256: b891718c424f6a69a55cbd4859c7b5a4adede618bde7c2bb7acd67af0ffa5931
```

Comment:
Appears the checksum is incorrect. 😕

ref: https://circleci.com/gh/conda-forge/pygpu-feedstock/78

Reply:
This is weird since libgpuarray builds passed and the use the same tag with the same hash.

Reply:
Unfortunately GitHub archives are not guaranteed to keep the same checksum. 😒 This is because it is autogenerated. They do perform some (unspecified) testing to avoid deviations, but they make no guarantees. Things like the order files are added to the archive or timestamp changes of files in the archive can result in this difference. We do periodically see issues like this, but not usually this soon after.

Reply:
I just confirmed that ef11ee6f8d62d53831277fd3dcab662aa770a5b5de2d30fe3018c4af959204da is the right hash (and it points to the right commit).

Reply:
Maybe I'll send GitHub another email about this.

In general this doesn't work, but may work for you since you are a maintainer of this library, is to upload your own tarballs and publish them as GitHub releases. That way you are guaranteed stable checksums as the tarball is static. There is a way to get Travis CI to do this for you too.

Reply:
I might just do that.  We already have working building conda packages for the regular conda and uploading them to our channel for tags.  Adding the upload of the archive shouldn't be too hard.

Reply:
Yeah, Travis CI has a [deploy provider]( https://docs.travis-ci.com/user/deployment/releases/ ) for it, which might make it easier.